src/engine/strat_engine/tests/util.rs

```diff
@@ -0,0 +1,60 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+use std::path::PathBuf;
+
+use devicemapper::{DM, DmFlags, DevId};
+
+use nix::mount::umount;
+use mnt::get_submounts;
+
+/// Remove all device mapper devices which have a name which matches the naming convention for
+/// a stratis created device.
+fn dm_stratis_devices_remove() {
+
+    let dm = DM::new().unwrap();
+
+    // TODO: Remove this for loop when we can utilize the DM_DEFERRED_REMOVE flag and not cause
```

Comment:
Change TODO inline to FIXME in header as:

```
FIXME: When remove_devices() can be called with DM_DEFERRED_REMOVE flag without causing a kernel oops
then this method will really remove _all_ devices.
```
You could also include a link to the bz.